ably-ios/ARTRest.m

```diff
@@ -92,6 +95,10 @@ - (instancetype)initWithKey:(NSString *) key {
     return [self initWithOptions:[[ARTClientOptions alloc] initWithKey:key]];
 }
 
+- (void)dealloc {
+    [self.logger debug:__FILE__ line:__LINE__ message:@"%p dealloc", self];
```

Comment:
dealloc is automatically called. The compiler do that work for you.

On 24 December 2015 at 18:53:51, Toni Cárdenas (notifications@github.com) wrote:

In ably-ios/ARTRest.m:

> @@ -92,6 +95,10 @@ - (instancetype)initWithKey:(NSString *) key {
>      return [self initWithOptions:[[ARTClientOptions alloc] initWithKey:key]];
>  }
>   
> +- (void)dealloc {
> +    [self.logger debug:__FILE__ line:__LINE__ message:@"%p dealloc", self];
Missing [super dealloc]?

—
Reply to this email directly or view it on GitHub.